packages/gatsby/index.d.ts

```diff
@@ -1283,29 +1283,6 @@ export type ActivityArgs = {
   id?: string
 }
 
-export interface Reporter {
-  stripIndent: (input: string) => string
-  format: object
-  setVerbose(isVerbose?: boolean): void
-  setNoColor(isNoColor?: boolean): void
-  panic: LogErrorType
-  panicOnBuild: LogErrorType
-  error: LogErrorType
-  uptime(prefix: string): void
-  success: LogMessageType
-  verbose: LogMessageType
-  info: LogMessageType
-  warn: LogMessageType
-  log: LogMessageType
-  activityTimer(name: string, activityArgs?: ActivityArgs): ActivityTracker
-  createProgress(
-    text: string,
-    total?: number,
-    start?: number,
-    activityArgs?: ActivityArgs
-  ): ProgressActivityTracker
-}
-
```

Comment:
Are those changes needed for this PR? those reporter type changes almost feel like should be separate PR?

Reply:
Done: https://github.com/gatsbyjs/gatsby/pull/25045

Reply:
Merged